### PR TITLE
akado.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/Akado.ru.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!--(www.)? refused
+popodpiske. mismatch
+forum. timed out
+mail. self signed
+graffiti. timed out
+speedtest. timed out
+radio. timed out
```
Comment (J0WI) [inline src/chrome/content/rules/Akado.ru.xml]: tv.akado.ru

Comment (galeksandrp) [inline src/chrome/content/rules/Akado.ru.xml]: Added.

File: src/chrome/content/rules/Akado.ru.xml [hunk 2]
```diff
@@ -1,14 +1,124 @@
...
+widget.tv.akado.ru ⁴
+web.akado.ru ³
+web2.akado.ru ³
+winterm01.akado.ru ³
+
+
+¹ mismatch
+² refused
+³ timed out
+⁴ self signed
```
Comment (J0WI) [inline src/chrome/content/rules/Akado.ru.xml]: Please double check for cert chain issues.

**Comment by J0WI** [comment]
ping @galeksandrp

**Pull request closed without merging**